test/jdk/java/util/Formatter/BasicDateTime.java

```diff
@@ -450,5 +452,24 @@ public static void test() {
         tryCatch("%%%", UnknownFormatConversionException.class);
         // perhaps an IllegalFormatArgumentIndexException should be defined?
         tryCatch("%<%", IllegalFormatFlagsException.class);
+
+        // %tF LocalDate
+        test("%tF", "2023-01-13", LocalDate.of(2023, 1, 13));
+        test("%tF", "2023-10-03", LocalDate.of(2023, 10, 3));
+        test("%tF", "0001-10-03", LocalDate.of(1, 10, 3));
+        test("%tF", "0012-10-03", LocalDate.of(12, 10, 3));
+        test("%tF", "0123-10-03", LocalDate.of(123, 10, 3));
+        test("%tF", "+12345-10-03", LocalDate.of(12345, 10, 3));
+        test("%tF", "-0001-10-03", LocalDate.of(-1, 10, 3));
+        test("%tF", "-0012-10-03", LocalDate.of(-12, 10, 3));
+        test("%tF", "-0123-10-03", LocalDate.of(-123, 10, 3));
+        test("%tF", "-1234-10-03", LocalDate.of(-1234, 10, 3));
+        test("%tF", "-12345-10-03", LocalDate.of(-12345, 10, 3));
+
+        Locale localeEuES = Locale.forLanguageTag("eu-ES");
+        test(localeEuES,
+                "%tF",
+                DecimalFormatSymbols.getInstance(localeEuES).getMinusSign() + "2023-01-13",
+                LocalDate.of(-2023, 1, 13));
```

Comment:
Needs some comments here, otherwise it would be a bit cryptic. Also, "eu-ES" locale using `\u2212` depends on the current CLDR implementation, so you might want to check all locales with  `Locale.availableLocale()`.

Reply:
While it might be reasonable to localize using `getMinusSign()` this will introduce a new inconsistency with `DateTimeFormatter` (which *does not* localize minus signs in front of years): 
```
int minus = java.text.DecimalFormatSymbols.getInstance(Locale.forLanguageTag("eu-ES")).getMinusSign()
minus ==> 8722

int first = DateTimeFormatter.ISO_DATE.withLocale(Locale.forLanguageTag("eu-ES")).format(ZonedDateTime.now().minus(
   ...> 4000, java.time.temporal.ChronoUnit.YEARS
   ...> )).charAt(0)
first ==> 45
```

Reply:
Within java.util.Formatter, the year formatter supports only the range [0,9999].
Only ISO_STANDARD_DATE has a ISO 8601 format defined for numbers outside that range. 
The formatting in java.time is defined over the full range of years.

Reply:
I think it is more reasonable for ISO_STANDARD_DATE to be consistent with java.time.DateTimeFormatter, ISO 8601 format is not localized.

Reply:
IIUC, what we are trying to fix here is to align the weird *behavior* outside of 0-9999 in java.util.Formatter with java.time's, not how they are represented. j.u.Formatter already formats numbers in local digits, so I believe it is natural to use the localized negative prefix as well.